docs/src/pages/en/responsive-controls.mdx

```diff
@@ -0,0 +1,441 @@
+---
+title: Responsive controls
+description: Learn how to make controls responsive with Media Chrome
+layout: ../../layouts/MainLayout.astro
+---
+
+import SandpackContainer from "../../components/SandpackContainer.astro";
+
+To provide the best user experience across the myriad of devices available in today's world, it's a good idea to modify your video player UI to adapt to different screen sizes.
```

Comment:
suggestion(non-blocking): Talk about player sizes, since folks can already use `@media` queries for screen size responsive design.